Source/WebKit/UIProcess/ios/forms/WKFileUploadPanel.mm

```diff
@@ -646,6 +646,25 @@ - (void)_dismissDisplayAnimated:(BOOL)animated
     return availableMediaTypeUTIs;
 }
 
+#if HAVE(PHOTOS_UI)
+- (PHPickerConfigurationAssetRepresentationMode)_preferredAssetRepresentationMode
+{
+    // FIXME (270470): `PHPickerConfigurationAssetRepresentationModeCurrent` should always be used
+    // and transcoding should be done based on the value of the `accept` attribute
+
+    if (![_delegate respondsToSelector:@selector(fileUploadPanelPhotoPickerPrefersOriginalImageFormat:)] || ![_delegate fileUploadPanelPhotoPickerPrefersOriginalImageFormat:self])
+        return PHPickerConfigurationAssetRepresentationModeCompatible;
+
+    if (![_acceptedUTIs count])
+        return PHPickerConfigurationAssetRepresentationModeCurrent;
+
+    if ([_acceptedUTIs containsObject:UTTypeImage.identifier])
```

Comment:
What if there's other UTIs in this set?

Reply:
`UTTypeImage.identifier` will only be present if someone specifies `image/*`. If there other items in the set, that would mean they did something like `accept="image/*,image/png"` or `accept="image/*,.docx"`.

In those cases, the site has still indicated that they support all image types.